Kickstarter-iOS/Features/PledgePaymentMethods/Views/Cells/PostCampaignPledgeRewardsSummaryCell.swift

```diff
@@ -39,17 +46,16 @@ final class PostCampaignPledgeRewardsSummaryCell: UITableViewCell, ValueCell {
 
     self.amountLabel.adjustsFontForContentSizeCategory = true
 
-    _ = self.rootStackView
-      |> rootStackViewStyle(self.traitCollection.preferredContentSizeCategory > .accessibilityLarge)
+    self.applyContainerStackViewStyle(self.containerStackView)
```

Comment:
This kind of naming convention works, too! I'm not sure we have enough style helpers that aren't in prelude to have a project-wide naming convention yet, but I would still lean toward `styleContainerStackView` and ect instead, just because it's a little shorter. Definitely not blocking, though!